feature-libs/customer-ticketing/core/facade/customer-ticketing.service.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@angular/core';
+import { CustomerTicketingFacade } from '@spartacus/customer-ticketing/root';
```

Comment:
this is complaining because this facade does not exist in root yet.

You can create a 'dummy/fake-for-now' facade to bypass this